src/main/java/com/pinterest/secor/util/orc/VectorColumnFiller.java

```diff
@@ -105,11 +104,21 @@ public void convert(JsonElement value, ColumnVector vect, int row) {
                 vect.noNulls = false;
                 vect.isNull[row] = true;
             } else {
-                TimestampColumnVector vector = (TimestampColumnVector) vect;
-                vector.set(
-                        row,
-                        Timestamp.valueOf(value.getAsString().replaceAll(
-                                "[TZ]", " ")));
+                if (value.getAsJsonPrimitive().isString()) {
+                    TimestampColumnVector vector = (TimestampColumnVector) vect;
+                    vector.set(
+                            row,
+                            Timestamp.valueOf(value.getAsString().replaceAll(
+                                    "[TZ]", " ")));
+                } else if (value.getAsJsonPrimitive().isNumber()) {
+                    TimestampColumnVector vector = (TimestampColumnVector) vect;
+                    vector.set(
+                            row,
+                            new Timestamp(value.getAsLong()));
+                } else {
+                    vect.noNulls = false;
```

Comment:
Do you want to issue some warnings in this case?  If you worried about too much logging being emitted, you can so some exponential back-off logging (e.g. logging on 1th, 2nd, 4th, 8th ... occurrences), those are very useful to debugging.

Reply:
@HenryCaiHaiying thanks for response. Yeah I did check the logging from VectorColumnFiller.java as well, but couldn't find any in that package path. I do not worry about too much logging because log would be only debug level (for only debugging as well). I'm going to try back-off logging as well 👍

Reply:
I think the problem is you don't any loggings when it gets to 'else' part of the code, usually this kind of logging is probably on warning level.  And to avoid generate too much warnings in the log file, you can try to log with exponential backoff.